apps/docs/content/docs/components/avatar.mdx

```diff
@@ -279,17 +279,21 @@ import { Avatar } from '@nextui-org/react';
 | **zoomed**       | `boolean`               | `true/false`                             | Zoomed avatar                          | `false`   |
 | **squared**      | `boolean`               | `true/false`                             | Squared avatar                         | `false`   |
 | **icon**         | `ReactNode`             | -                                        | Show icon in avatar                    | -         |
+| **css**          | `styleObject`           | `css.properties`                         | Override Default CSS style             | -         |
+| **as**           | `ElementType`           | `'Button', 'a'`                          | Changes which tag component outputs    | `Avatar`  |
```

Comment:
Could you please change the `as` type` to `keyof JSX.IntrinsicElements` and `accepted` values to `-`?, apply this to all components docs 🙏🏻